### PR TITLE
Adding support for creating a bunch of events for an given query

### PR DESCRIPTION
**Pull request by diggyk**
We need a way to easily create a bunch of events for a given external query.  This work basically adds that support into the client and backing API.  If a query is specified, it gets hosts for that query and then creates a bunch of events in batch.

To speed up labor processing, we only throw one slack message at the end of that loop.  And we only show the actual labors if there is a reasonable number of labors closed.

In the Events model, I've also changed the tx column to be a BigInteger.  I'm not sure if that's necessary.  Originally, events would only be created en masse if part of a quest creation so the quest id was used to ensure sufficient uniqueness.  Since events can be created en masse in other ways now, I need to make sure I can generate a unique ID for this "transaction ID" to make sure I can accurately read back the values I wrote in the mass create.  Comments on this and all other parts welcome.

### CODE REVIEW

**Comment by diggyk** [comment]
To note, this is not going to go into production, but rather to the dev server so I can test to see if I need to further optimize the updating of Labors or if the current loop will be fast enough.

**Comment by diggyk** [comment]
Fixing :)

**Comment by diggyk** [comment]
Fixed!

File: hermes/handlers/api.py [hunk 1]
```diff
@@ -857,26 +880,82 @@ def post(self):
...
+            Host.create_many(self.session, new_hosts_needed)
+            hosts = (
+                self.session.query(Host).filter(
+                    Host.hostname.in_(hostnames)
+                ).all()
+            )
 
-        if host is None:
-            host = Host.create(self.session, hostname)
+        if len(hosts) == 0:
```
Comment (gmjosack) [inline hermes/handlers/api.py:908]: This is usually written as `if not hosts` which would match both the `None` case and the `[]` case.

Comment (diggyk) [inline hermes/handlers/api.py:908]: Good note.  I knew that and dont know why I didn't use it.

**Pull request merged**